src/Service/ServiceExe.cs

```diff
@@ -1514,13 +1514,13 @@ public void Start(bool forceUserTaskRerun)
             // Start IPC server on port 45684. Only start after all boot tasks are complete.
             if (EnableNetworkAccess && !DisableNetworkAccess) 
             {
-                FOServiceExe.ipcHost = new IpcServiceHostBuilder(FOServiceExe.ipcSvcProvider).AddTcpEndpoint<IFactoryOrchestratorService>("tcp", IPAddress.Any, 45684)
+                FOServiceExe.ipcHost = new IpcServiceHostBuilder(FOServiceExe.ipcSvcProvider).AddTcpEndpoint<IFactoryOrchestratorService>("tcp", IPAddress.Any, 45684, new JKang.IpcServiceFramework.Tcp.TcpConcurrencyOptions(5))
```

Comment:
if a 6th client tries, does it just get put behind a retry?

Reply:
The API they call just blocks until a slot is available. No timeout or rerty needed.

Reply:
It uses a semaphone lock